lib/asciidoctor/bipm/converter.rb

```diff
@@ -174,7 +174,7 @@ def committee_validate(xml)
 
       def boilerplate_file(xmldoc)
         return super unless xmldoc&.at("//bibdata/ext/editorialgroup/committee/@acronym")&.value == "JCGM"
-        baselocation("lib/asciidoctor/bipm/boilerplate-jcgm-en.xml")
+        File.join(File.dirname(__FILE__), "boilerplate-jcgm-en.xml")
```

Comment:
@opoudjis I noted that `baselocation` method just return passed value after https://github.com/metanorma/metanorma-generic/commit/b2e274bfb856a321940daf193682905876f920dc commit. Is that expected?